infra/aws/README.md

```diff
@@ -0,0 +1,83 @@
+Building a cluster on AWS
+=========================
+
+This directory containes three subdirectories:
+
+* packer - used to build the base image
+* terraform - used to spin up EC2 virtual machines
+* ansible - used to configure the virtual machines
+
+# Packer
+
+The packer directory contains a Hashicorp Packer template used
+to build the base AMI disk image for AWS. 
+
+If you are using the Protocol Labs Test Infra AWS account, the
+AMI image has already been built and published.
+
+See [packer.io](https://www.packer.io/) for information on how
```

Comment:
Do people need learn Packer to use the cluster?

Reply:
No. But it's good to know how the image is built and where to modify it.